feat(contacts): add contacts component - INNO-368

kalinchernev opened the pull request.

emeryro commented on src/framework/atoms/button/styles/button.less:
```diff
@@ -17,3 +17,7 @@
   background: @brand-info;
 }
+
+.btn-full {
```
There is a class in Bootstrap already available for this: .btn-block

emeryro commented on src/framework/molecules/contacts/templates/contacts.html:
```diff
@@ -0,0 +1,55 @@
+      <li typeof="skos:Concept" property="rdfs:label skos:prefLabel" >English</li>
+    </ul>
+  </div>
+  <div class="contacts__pane user-profile-block-social-media">
```
Maybe icons should be a little large. In theory we should have some modifiers to make text larger/smaller, but here I guess one line of css is enough for these icons.

kalinchernev replied:
Agree for both.

emeryro commented on src/SUMMARY.md:
```diff
@@ -24,7 +24,7 @@
         *   [✓ Community create button](./framework/molecules/community-create-button/README.md)
         *   [✓ Community navigation](./framework/molecules/community-navigation/README.md)
-        *   [✗ Contacts](./framework/molecules/contacts/README.md)
+        *   [✓ Contacts](./framework/molecules/contacts/README.md)
```
Do not forget to change it in readme file too ;)

kalinchernev commented:
Changed the icons without too much abstraction. Added also a small hover effect, because @degliwe said once we have to make small things here and there which make us happy :) (If you don't like it, please tell me)
